ui/src/tool-panel/tool-panel.tsx

```diff
@@ -92,6 +94,17 @@ type ToolPanelTab = {
 const isSameAsset = (a: ToolPanelAsset | undefined, b: ToolPanelAsset | undefined) =>
     !!a && !!b && a.id === b.id && a.type === b.type;
 
+const linkingStateLayoutAlignmentTabType = (type: MapAlignmentType) => {
+    switch (type) {
+        case MapAlignmentType.LocationTrack:
+            return 'LOCATION_TRACK';
+        case MapAlignmentType.ReferenceLine:
+            return 'TRACK_NUMBER';
```

Comment:
Geometria on käsitteellisesti pituusmittauslinjoilla, mutta meidän toolpanel näyttää ratanumeroita, hence the switcheroo